MonoGame.Framework/SDL/SDLGamePlatform.cs

```diff
@@ -129,13 +129,17 @@ private void SdlRunLoop()
                     if (!_keys.Contains(key))
                         _keys.Add(key);
                     char character = (char)ev.Key.Keysym.Sym;
+                    if (KeysHelper.IsKey((int)key))
```

Comment:
KeyboardUtil.ToXna already returns a valid key enum so this will always return true.